src/index.js

```diff
@@ -195,7 +195,12 @@ export default function (babel) {
       if (!tailPath) continue;
 
       if (tailPath.isExpressionStatement()) {
-        tailPath.replaceWith(getNewNode(tailPath.node.expression, tailPath));
+        const expression = tailPath.node.expression;
+        if (t.isAwaitExpression(expression)) {
```

Comment:
I moved it since that matches other uses, like `ObjectComprehensions`.

@rattrayalex you want a comment added there that it only applies to implicits?